vagrant/playbooks/host-pulp2.yml

```diff
@@ -0,0 +1,61 @@
+---
+# host-pulp2.yml
+# Additional tasks to host pulp2 Docker container box on top of the pulp3 VM box.
+
+- hosts: all
+  become: true
+  vars:
+    paths_to_vars_files:
+      - '../../local.{{ ansible_hostname }}-config.yml'
+      - '../../local.dev-config.yml'
+      - '../../example.dev-config.yml'
+  pre_tasks:
+    - name: Load local config
+      include_vars: "{{ item }}"
+      with_first_found: "{{ paths_to_vars_files }}"
+
+    # The version string below is the highest of all those in roles' metadata:
+    # "min_ansible_version". It needs to be kept manually up-to-date.
+    - name: Verify Ansible meets min required version
+      assert:
+        that: "ansible_version.full is version_compare('2.9', '>=')"
+        msg: >
+          "You must update Ansible to at least 2.9 to use this version of Pulp 3 Installer."
+  roles:
+    - role: geerlingguy.docker
+      vars:
+        docker_users:
+          - "{{ developer_user }}"
+          - vagrant
```

Comment:
cool!

Reply:
Yeah, this is one way to pass vars to a particular role.